spec/get_player_input_spec.rb

```diff
@@ -0,0 +1,45 @@
+require './lib/get_player_input.rb'
+require 'stringio'
+
+
+describe "#get_user_input" do
+    let(:board){Array.new(3){Array.new(3, "_")}}
+    it "should record player X's input in the board" do
```

Comment:
This is a very good test for a number of reasons, but also note how the nouns and verbs relate directly to playing a game of Tic Tac Toe.  You could imagine translating the code in this test to natural language, and being able to use it to show someone an example of how a game of Tic Tac Toe is played.

As you get into bigger and bigger projects, you may notice a lot of code that contains nouns having very little to do with the application or its "problem domain".  Balancing the necessary code we're all interested in with the other bits that are necessary to do the job is part of maintaining a clean architecture.